src/Conversion/IConverter.cs

```diff
@@ -513,22 +513,22 @@ protected GarminDeviceInfo GetDeviceInfo()
 			}
 
 			return info;
-		}
-
-		protected DistanceUnit GetDistanceUnit(string unit)
-		{
-			switch (unit.ToLower())
-			{
-				case "km":
-					return DistanceUnit.Kilometers;
-				case "mi":
-					return DistanceUnit.Miles;
-				case "ft":
-					return DistanceUnit.Feet;
-				default:
-					Log.Debug("Found unkown distance unit {@Unit}", unit);
-					return DistanceUnit.Unknown;
-			}
-		}
-	}
-}
+		}
+
+		protected DistanceUnit GetDistanceUnit(string unit)
+		{
+			switch (unit?.ToLower())

```

Comment:
All other changes in this file are spaces->tabs. This line is the only functional change.